heroku_connect/db/models/base.py

```diff
@@ -238,3 +253,19 @@ def check(cls, **kwargs):
         errors.extend(cls._check_unique_sf_field_names())
         errors.extend(cls._check_upsert_field())
         return errors
+
+    def delete(self, using=None, keep_parents=False):
```

Comment:
Please use `**kwargs` to be compatible with future changes. (Since you are not using any of the arguments.